card/mod/core/lib/tasks/card.rake

```diff
@@ -10,6 +10,8 @@ namespace :card do
   desc "Runs migrations, installs mods, and updates symlinks"
   task :update do
     failing_loudly "decko update" do
+      Rake::Task["decko:repo:update"].invoke if Rake::Task.task_defined?("decko:repo:update")
+
```

Comment:
I'm happy to approve this. But what are your thoughts on this solution?

https://github.com/guillermo/rake-hooks

Reply:
@xithan ?

Reply:
I tried it and I like it